test/local/statsd_tests.js

```diff
@@ -51,7 +51,7 @@ test(
           t.equal(name, 'fxa.auth.some-event')
           t.equal(value, 1)
           t.ok(sampleRate)
-          t.notOk(tags)
+          t.ok(tags)
```

Comment:
@vbudhram Yeap that would work too! We already do this here: https://github.com/mozilla/fxa-auth-server/blob/b8900eca7c0dbd4ceef63ae205a1ecce1201c6a2/test/local/db_tests.js#L104

Up to you if you want to validate the size of the array or just use isArray